---
layout: subsite-galaxy
website: https://singlecell.usegalaxy.eu
gitter: usegalaxy-eu/Lobby
subdomain: singlecell
---

![single cell Galaxy](/assets/media/logo_single_cell.svg){:.sc-intro-left}

# Welcome to the world of Single Cell Omics
{:.no_toc}

The Single Cell Omics workbench is a comprehensive set of analysis tools and consolidated workflows. The workbench is based on the [Galaxy framework](https://galaxyproject.org){:target="_blank"},
which guarantees simple access, easy extension, flexible adaption to personal and security needs, and sophisticated analyses independent of command-line knowledge.

The current implementation comprises more than 20 bioinformatics tools dedicated to different research areas of single cell biology.

This service is a joint project between different groups from the [Earlham Institute](http://www.earlham.ac.uk){:target="_blank"}, the [EMBL-EBI](https://www.ebi.ac.uk){:target="_blank"}, [EMBL](https://gbcs.embl.de){:target="_blank"} the [Sorbonne University](http://artbio.fr){:target="_blank"}, [Peter MacCallum Cancer Centre](https://www.petermac.org){:target="_blank"} and the [University of Freiburg](https://galaxyproject.eu/freiburg/){:target="_blank"}.
The server is part if the European Galaxy server and is maintained by the [RNA Bioinformatics Center (RBC)](https://www.denbi.de/network/rna-bioinformatics-center-rbc){:target="_blank"} as part of [de.NBI](https://www.denbi.de){:target="_blank"} and [ELIXIR](http://elixir-europe.org){:target="_blank"}.


# Content
{:.no_toc}

1. TOC
{:toc}


# Get started

Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take [a guided tour]({{ page.website }}/tours/core.galaxy_ui){:target="_blank"} through Galaxy's user interface.

# Training and Workshops

**Want to learn more about single cell analyses? Check out the following hands-on tutorials from [the Galaxy Training Network](https://galaxyproject.github.io/training-material/){:target=_"blank"}**, or come to one of our workshops.

We are passionate about training. So we are working in close collaboration with the [Galaxy Training Network (GTN)](https://galaxyproject.org/teach/gtn/){:target="_blank"} to develop training materials of data analyses based on Galaxy {% cite batut2017community %}. These materials hosted on the GTN GitHub repository are available online at [https://training.galaxyproject.org](https://training.galaxyproject.org){:target="_blank"}.

<!-- ## Workshops -->

<!-- We will be hosting a scRNA-seq workshop [this March](https://usegalaxy-eu.github.io/event/2020-01-20-GalaxyWS_scrna_FR/plain.html). Please register whilst spots are still open! -->

## Material


| Lesson                                          | Slides                                                                                                                                                                                                   | Hands-on                                                                                                                                                                                          | Input dataset                                                                                           | Workflows                                                                                                                                                                                                                                                 | Galaxy History                                                                                                                                                                                                                                                               |
| ---                                             | ---                                                                                                                                                                                                      | ---                                                                                                                                                                                               | ---                                                                                                     | ---                                                                                                                                                                                                                                                       | ---                                                                                                                                                                                                                                                                          |
| Introduction to Transcriptomics                 | [<i class="fa fa-slideshare" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/transcriptomics/slides/introduction.html){:target="_blank"}                               |                                                                                                                                                                                                   |                                                                                                         |                                                                                                                                                                                                                                                           |                                                                                                                                                                                                                                                                              |
| Plates, Batches, and Barcodes                   | [<i class="fa fa-slideshare" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html){:target="_blank"} |                                                                                                                                                                                                   |                                                                                                         |                                                                                                                                                                                                                                                           |                                                                                                                                                                                                                                                                              |
| Understanding Barcodes                          |                                                                                                                                                                                                          | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-umis/tutorial.html){:target="_blank"}               | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/2573177){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>]({{ page.website }}/workflows/run?id=d7aa4c258e2edc95){:target="_blank"}                                                                                                                              | [<i class="fa fa-list-ul" aria-hidden="true"></i>]({{ page.website }}/u/mehmet-tekman/h/understanding-barcodes){:target="_blank"}                                                                                                                                            |
| Pre-processing of Single-Cell RNA Data          |                                                                                                                                                                                                          | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.html){:target="_blank"}      | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/3253142){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>]({{ page.website }}/workflows/run?id=49073a24429d93d6){:target="_blank"} [<i class="fa fa-share-alt" aria-hidden="true"></i>]({{ page.website }}/workflows/run?id=76a6330d5fc241c7){:target="_blank"} | [<i class="fa fa-list-ul" aria-hidden="true"></i>]({{ page.website }}/u/mehmet-tekman/h/celseq2-single-batch-mm10){:target="_blank"} [<i class="fa fa-list-ul" aria-hidden="true"></i>]({{ page.website }}/u/mehmet-tekman/h/celseq2-multi-batch-workflow){:target="_blank"} |
| Pre-processing of 10X Single-Cell RNA Datasets  |                                                                                                                                                                                                          | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-preprocessing-tenx/tutorial.html){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/3457880){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>]({{ page.website }}/workflows/run?id=d79309343e2a5d62){:target="_blank"}                                                                                                                              | [<i class="fa fa-list-ul" aria-hidden="true"></i>]({{ page.website }}/u/mehmet-tekman/h/pre-processing-of-10x-single-cell-rna-datasets-1){:target="_blank"}                                                                                                                  |
| Single-Cell Quality Control with Scater         |                                                                                                                                                                                                          | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-scater-qc/tutorial.html){:target="_blank"}          | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/3386291){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>]({{ page.website }}/workflows/run?id=f055b8fa294d4be8){:target="_blank"}                                                                                                                              | [<i class="fa fa-list-ul" aria-hidden="true"></i>]({{ page.website }}/u/mehmet-tekman/h/single-cell-quality-control-with-scater){:target="_blank"}                                                                                                                           |
| Downstream Single-cell RNA analysis with RaceID |                                                                                                                                                                                                          | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-raceid/tutorial.html){:target="_blank"}             | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/1511582){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>]({{ page.website }}/workflows/run?id=2f8d9fb85242eca7){:target="_blank"}                                                                                                                              | [<i class="fa fa-list-ul" aria-hidden="true"></i>]({{ page.website }}/u/mehmet-tekman/h/raceid-training-material){:target="_blank"}                                                                                                                                          |
| Clustering 3K PBMCs with ScanPy                 |                                                                                                                                                                                                          | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.html){:target="_blank"}      | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/3581213){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>]({{ page.website }}/workflows/run?id=921cab3e6faf30be){:target="_blank"}                                                                                                                              | [<i class="fa fa-list-ul" aria-hidden="true"></i>]({{ page.website }}/u/mehmet-tekman/h/clustering-3k-pbmcs-with-scanpy){:target="_blank"}                                                                                                                                   |
{:.table.table-striped}


# Available tools

In this section we list all tools that have been integrated in the the single cell workbench. The list is likely to grow as soon as further tools and workflows are contributed. To ease readability, we divided them into categories.

## Preprocessing

|                                                                                                                                                                                                                                                                 | Formats     | QC                          | Demultiplexing              | Mapping                                      | Quantification              | References                                                                                                                                                                   |
|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-----------:|:---------------------------:|:---------------------------:|:--------------------------------------------:|:---------------------------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
|                                        [AnnData / Loom Suite](#suites-mentioned "Import, Export, Modify, Inspect and Convert AnnData files. Converts between various different single-cell formats and permits various levels of inspection and manipulation.") | CSV LOOM H5 |                             |                             |                                              |                             | [AnnData Documentation](https://anndata.readthedocs.io/en/latest/)                                                                                                           |
|                    {% include tool.html id="crosscontamination_barcode_filter" label="Cross Contamination Barcode Filter" title="QC tool for inspecting raw count matrices and determining the level of contamination between neighbouring sequencing wells" %} | CSV         | <i class="fa fa-check"></i> |                             |                                              |                             | [Freiburg Galaxy Team, 2019](https://galaxyproject.eu/freiburg/people.html)                                                                                                  |
|                                                                                                                                                          {% include tool.html id="fastp" label="Fastp" title="Fast all-in-one preprocessing for FASTQ files" %} | FASTA/Q     | <i class="fa fa-check"></i> |                             |                                              |                             | [Shifu et al. 2018](https://doi.org/10.1101/274100){:target="_blank"}                                                                                                        |
| {% include tool.html id="fastqc" label="FastQC" title="A quality control tool for high throughput sequence data" %} / {% include tool.html id="multiqc" label="MultiQC" title="MultiQC aggregates results from bioinformatics analyses into a single report" %} | FASTQ       | <i class="fa fa-check"></i> |                             |                                              |                             | [Ewels et al. 2016](https://doi.org/10.1093/bioinformatics/btw354){:target="_blank"}                                                                                         |
|                                                                                                                                                       {% include tool.html id="htseq-count" label="htseq-count" title="Tool for counting reads in features" %} | BAM         |                             |                             |                                              | <i class="fa fa-check"></i> | [Anders et al. 2015](https://doi.org/10.1093%2Fbioinformatics%2Fbtu638){:target="_blank"}                                                                                    |
|                                                                                                                                         [Je Suite](#suites-mentioned "A versatile suite to handle multiplexed NGS libraries with unique molecular identifiers") | FASTA/Q BAM | Filtering                   | <i class="fa fa-check"></i> |                                              | <i class="fa fa-check"></i> | [Girardot at al. 2016](https://doi.org/10.1186/s12859-016-1284-2)                                                                                                            |
|              [UMI-tools Suite](#suites-mentioned "Includes utilities to count, deduplicate, extract, group, and whitelist Unique Molecular Identifiers (UMIs). Modelling sequencing errors in Unique Molecular Identifiers to improve quantification accuracy") | FASTQ       | Filtering                   | <i class="fa fa-check"></i> |                                              | <i class="fa fa-check"></i> | [Smith et al. 2017](https://doi.org/10.1101/gr.209601.116){:target="_blank"}                                                                                                 |
|                                                                                  {% include tool.html id="RNA STAR" label="RNA STAR" title="Rapid spliced aligner for RNA-seq data, performs single-cell quantification only when used with *featureCounts*" %} | FASTA/Q     | Filtering                   |                             | <i class="fa fa-check"></i>                  | <i class="fa fa-check"></i> | [Dobin et al. 2013](https://academic.oup.com/bioinformatics/article/29/1/15/272537/STAR-ultrafast-universal-RNA-seq-aligner){:target="_blank"}                               |
|                                                                                                         {% include tool.html id="RNA STAR Solo" label="RNA STAR Solo" title="Supports Droplet (Drop-seq, 10X) protocols, and can emulate CellRanger pipeline" %} | FASTA/Q     | Filtering                   | <i class="fa fa-check"></i> | <i class="fa fa-check"></i>                  | <i class="fa fa-check"></i> | [Dobin et al. 2013](https://academic.oup.com/bioinformatics/article/29/1/15/272537/STAR-ultrafast-universal-RNA-seq-aligner){:target="_blank"}                               |
|                                                                                                                                                                  {% include tool.html id="bowtie2" label="Bowtie2" title="Fast and sensitive read alignment" %} | FASTA/Q     | Filtering                   |                             | <i class="fa fa-check"></i>                  |                             | [Langmead et al. 2012](https://doi.org/10.1038/nmeth.1923){:target="_blank"}                                                                                                 |
|                                                                                                                                           {% include tool.html id="hisat2" label="HISAT2" title="Hierarchical indexing for spliced alignment of transcripts" %} | FASTA/Q     | Filtering                   |                             | <i class="fa fa-check"></i>                  |                             | [Pertea et al. 2016](https://doi.org/10.1038/nprot.2016.095){:target="_blank"}                                                                                               |
|                                                                                                                                         {% include tool.html id="sailfish" label="Sailfish" title="Rapid Alignment-free Quantification of Isoform Abundance" %} | FASTA/Q BAM | Filtering                   | <i class="fa fa-check"></i> | <i class="fa fa-check"></i>                  | <i class="fa fa-check"></i> | [Patro et al. 2014](http://dx.doi.org/10.1038/nbt.2862){:target="_blank"}                                                                                                    |
|                                                                                         {% include tool.html id="alevin" label="Alevin" %} / {% include tool.html id="salmon" label="Salmon" title="Fast, accurate and bias-aware transcript quantification" %} | FASTA/Q BAM | Filtering                   |                             | Salmon                                       | Salmon                      | [Patro et al. 2017](http://dx.doi.org/10.1038/nmeth.4197){:target="_blank"}                                                                                                  |
|                                                                                              {% include tool.html id="scpipe" label="scPipe" title="Preprocessing pipeline for single cell RNA-seq. Also performs downstream analysis via Scater and SCRAN." %} | FASTQ BAM   | Filtering                   | <i class="fa fa-check"></i> | Subread                                      | Subread                     | [Tian et al. 2018](https://doi.org/10.1371/journal.pcbi.1006361){:target="_blank"}                                                                                           |
|                                                                                                                                      {% include tool.html id="featurecounts" label="FeatureCounts" title="Ultrafast and accurate read summarization program" %} | BAM         |                             |                             | Subread                                      | Subread                     | [Liao et al. 2014](http://dx.doi.org/10.1093/bioinformatics/btt656){:target="_blank"}                                                                                        |
|                                                                                                                      {% include tool.html id="bwa" label="BWA" title="Software package for mapping low-divergent sequences against a large reference genome" %} | FASTA/Q     |                             |                             | <i class="fa fa-check"></i> not splice-aware |                             | [Li and Durbin 2009](https://doi.org/10.1093/bioinformatics/btp324){:target="_blank"}, [Li and Durbin 2010](https://doi.org/10.1093/bioinformatics/btp698){:target="_blank"} |
{: .table.table-striped .tooltable}


## Downstream

|                                                                                                                                                                          | Formats                              | Filtering                   | Normalisation                              | Batch / Confounder Removal         | Clustering                  | Embedding                           | Lineage/Pseudotime | Classification / Marker     | Description                                                                    |
|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:------------------------------------:|:---------------------------:|:------------------------------------------:|:----------------------------------:|:---------------------------:|:-----------------------------------:|:------------------:|:---------------------------:|:-------------------------------------------------------------------------------|
|                                                                                        [SC3 Suite](#suites-mentioned "Consensus clustering of single-cell RNA-Seq data") | SCE                                  |                             |                                            |                                    | Consensus K-means           | PCA                                 |                    |                             | [Kiselev et al](https://doi.org/10.1038/nmeth.4236)                            |
|                                                                   [ScanPy Suite](#suites-mentioned "The first scRNA pipeline written in Python with native 10X support") | AnnData CSV LOOM excel mtx umi-tools | <i class="fa fa-check"></i> | CPM                                        | Cell Cycle ComBat MNNCorrect BBKNN | PCA leiden louvain          | PCA tSNE UMAP                       | dendrogram PAGA    | <i class="fa fa-check"></i> | [Wolf at al. 2018](https://doi.org/10.1186/s13059-017-1382-0)                  |
|                                    [Scater Suite](#suites-mentioned "Pre-processing, quality control, normalization and visualization of single-cell RNA-seq data in R") | SCE SC3                              | <i class="fa fa-check"></i> | CPM FPKM TPM                               |                                    | PCA                         | PCA tSNE UMAP PHATE diffmap         |                    |                             | [McCarthy et al. 2017](https://doi.org/10.1093/bioinformatics/btw777)          |
|                                                  [Seurat Suite](#suites-mentioned "An R package designed for QC, analysis, and exploration of single-cell RNA-seq data") | CSV 10X H5 Alevin                    | <i class="fa fa-check"></i> | Log-Normalise Centered-log Relative-counts | SCTransform SNNAnchor              | SharedNN                    | PCA tSNE UMAP                       |                    | <i class="fa fa-check"></i> | [Satija et al. 2015](https://doi.org/10.1038/nbt.3192)                         |
|                                                [RaceID Suite](#suites-mentioned "Clustering using RaceID performs clustering, outlier detection, dimensional reduction") | CSV SCSeq                            | <i class="fa fa-check"></i> | Library Size                               | Linear CCcorrect (Cell cycle)      | k-means k-medians hclust    | PCA tSNE UMAP Fruchterman-Rheingold | StemID FateID      |                             | [Herman et al. 2018](https://doi.org/10.1038/nmeth.4662)                       |
| {% include tool.html id="table_compute" label="Table Compute" title="A tool to perform complex operations upon single or multiple tables using the pandas framework." %} | CSV                                  | <i class="fa fa-check"></i> | user-defined function                      | user-defined function              |                             |                                     |                    |                             | [Freiburg Galaxy Team, 2019](https://galaxyproject.eu/freiburg/people.html)     |
|                             {% include tool.html id="deseq2" label="DESeq2" title="Differential gene expression analysis based on the negative binomial distribution" %} | CSV htseq-count summarizedexperiment | <i class="fa fa-check"></i> | DESeq2                                     |                                    | <i class="fa fa-check"></i> |                                     |                    |                             | [Love et al. 2014](http://doi.org/10.1186/s13059-014-0550-8){:target="_blank"} |
|                        {% include tool.html id="scpipe" label="scPipe" title="CelSeq2, etc. Uses Scater and SCRAN for downstream, that also performs pre-processing." %} | CSV AnnData                          | <i class="fa fa-check"></i> | <i class="fa fa-check"></i>                | <i class="fa fa-check"></i>        | <i class="fa fa-check"></i> | <i class="fa fa-check"></i>         |                    |                             | [Tian et al. 2018](https://doi.org/10.1371/journal.pcbi.1006361)               |
{: .table.table-striped .tooltable}
<!-- |                                                                                                                                                                                                                                                            Monocle Suite: {% include tool.html id="monocle3_create" %} {% include tool.html id="monocle3_diffexp" %} {% include tool.html id="monocle3_learngraph" %} {% include tool.html id="monocle3_ordercells" %} {% include tool.html id="monocle3_partition" %} {% include tool.html id="monocle3_plotcells" %} {% include tool.html id="monocle3_preprocess" %} {% include tool.html id="monocle3_reducedim" %} | CSV AnnData                          |<i class="fa fa-check"></i>        | Library Size                               |<i class="fa fa-check"></i>                                 | Louvain KNN GMM          | ICA tSNE L1-graph SimplePPT DDRTree |<i class="fa fa-check"></i>                 |<i class="fa fa-check"></i>                      | An analysis toolskit for single-cell RNA-seq that works well with 10X [Trapnell et al. 2014](https://doi.org/10.1038/nbt.2859)                                          | -->


## Suites Mentioned

|     Suite Name | Tools                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
|---------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| AnnData / Loom | {% include tool.html id="anndata_export" label="Export" %}, {% include tool.html id="anndata_import" label="Import" %}, {% include tool.html id="anndata_inspect" label="Inspect" %}, {% include tool.html id="anndata_manipulate" label="Manipulate" %}, {% include tool.html id="modify_loom" label="Modify Loom" %}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
|             Je | {% include tool.html id="je_clip" label="Clip" %}, {% include tool.html id="je_demultiplex" label="Demultiplex" %}, {% include tool.html id="je_demultiplex_illu" label="Demultiplex Illumina" %}, {% include tool.html id="je_markdupes" label="MarkDuplicates" %}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
|      UMI-tools | {% include tool.html id="umi_tools_count" label="Count" %}, {% include tool.html id="umi_tools_dedup" label="Deduplicate" %}, {% include tool.html id="umi_tools_extract" label="Extract" %}, {% include tool.html id="umi_tools_group" label="Group" %}, {% include tool.html id="umi_tools_whitelist" label="Whitelist" %}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
|            SC3 | {% include tool.html id="sc3_calc_consens" label="Calculate Consensus" %}, {% include tool.html id="sc3_prepare" label="Prepare" %}, {% include tool.html id="sc3_calc_transfs" label="Calculate Transformations" %}, {% include tool.html id="sc3_calc_biology" label="DiffExp" %}, {% include tool.html id="sc3_estimate_k" label="Estimate" %}, {% include tool.html id="sc3_calc_dists" label="Calculate Distances" %}, {% include tool.html id="sc3_kmeans" label="K-means" %}                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
|         ScanPy | {% include tool.html id="scanpy_read_10x" label="Read 10X" %}, {% include tool.html id="scanpy_scale_data" label="Scale Data" %}, {% include tool.html id="scanpy_find_cluster" label="Find Cluster" %}, {% include tool.html id="scanpy_run_pca" label="Run PCA" %}, {% include tool.html id="scanpy_run_tsne" label="Run tSNE" %}, {% include tool.html id="scanpy_run_umap" label="Run UMAP" %}, {% include tool.html id="scanpy_compute_graph" label="Compute Graph" %}, {% include tool.html id="scanpy_find_variable_genes" label="Find Variable Genes" %}, {% include tool.html id="scanpy_normalise_data" label="Normalise Data" %}, {% include tool.html id="scanpy_filter_genes" label="Filter genes" %}, {% include tool.html id="scanpy_find_markers" label="Find Markers" %}, {% include tool.html id="scanpy_filter_cells" label="Filter Cells" %}, {% include tool.html id="scanpy_parameter_iterator" label="Parameter Iterator" %} |
|         Scater | {% include tool.html id="scater_create_qcmetric_ready_sce" label="Calculate QC Metrics" %}, {% include tool.html id="scater_filter" label="Filter" %}, {% include tool.html id="scater_normalize" label="Normalize" %}, {% include tool.html id="scater_plot_dist_scatter" label="Plot Scatter" %}, {% include tool.html id="scater_plot_exprs_freq" label="Plot Expression Frequency" %}, {% include tool.html id="scater_plot_pca" label="Plot PCA" %}, {% include tool.html id="scater_plot_tsne" label="Plot tSNE" %}                                                                                                                                                                                                                                                                                                                                                                                                                           |
|         Seurat | {% include tool.html id="seurat_read10x" label="Read 10X" %}, {% include tool.html id="seurat_run_pca" label="Run PCA" %}, {% include tool.html id="seurat_scale_data" label="Scale Data" %}, {% include tool.html id="seurat_filter_cells" label="Filter Cells" %}, {% include tool.html id="seurat_find_markers" label="Find Markers" %}, {% include tool.html id="seurat_dim_plot" label="Dim. Plot" %}, {% include tool.html id="seurat_normalise_data" label="Normalise Data" %}, {% include tool.html id="seurat_create_seurat_object" label="Create Seurat Object" %}, {% include tool.html id="seurat_export_cellbrowser" label="Export CellBrowser" %}, {% include tool.html id="seurat_run_tsne" label="Run tSNE" %}, {% include tool.html id="seurat_find_variable_genes" label="Find Variable Genes" %}, {% include tool.html id="seurat_find_clusters" label="Find Clusters" %}                                                        |
|         RaceID | {% include tool.html id="raceid_clustering" label="Clustering" %}, {% include tool.html id="raceid_filtnormconf" label="Filter Normalize Confounder" %}, {% include tool.html id="raceid_trajectory" label="Trajectory" %}, {% include tool.html id="raceid_inspectclusters" label="Inspect Clusters" %}, {% include tool.html id="raceid_inspecttrajectory" label="Inspect Trajectory" %}                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
{: .table.table-striped .tooltable}


<!-- BEGIN: The following tools were left out: --->
<!-- {% include tool.html id="STAR-fusion" %} | Fast fusion gene finder | [Haas et al. 2017](https://www.biorxiv.org/content/early/2017/03/24/120295){:target="_blank"} -->
<!-- ## Utilities -->
<!-- SAMtools | Utilities for manipulating alignments in the SAM format | [Heng et al. 2009](https://doi.org/10.1093/bioinformatics/btp352){:target="_blank"} -->
<!-- BEDTools | Utilities for genome arithmetic | [Quinlan and Hall 2010](https://doi.org/10.1093/bioinformatics/btq033){:target="_blank"} -->
<!-- deepTools | Tools for exploring deep-sequencing data | [Ramirez et al. 2014](https://doi.org/10.1093/nar/gku365){:target="_blank"}, [Ramirez et al. 2016](https://doi.org/10.1093/nar/gkw257){:target="_blank"} -->
<!-- END --->

# Contributors

- [Bérénice Batut](https://github.com/bebatut)
- [Mehmet Tekman](https://github.com/mtekman)
- [Alex Ostrovsky](https://github.com/astrovsky01)
- [Maria Doyle](https://github.com/mblue9)
- [Pablo Moreno](https://github.com/pcm32)
- [Ni Huang](https://github.com/nh3)
- [Jonathan Manning](https://github.com/pinin4fjords)
- [Christophe Antoniewski](https://github.com/drosofff)
- [Lea Bellenger](https://github.com/bellenger-l)
- [Jelle Scholtalbers](https://github.com/scholtalbers)
- [Charles Girardot](https://github.com/cgirardot)
- [Nicola Soranzo](https://github.com/nsoranzo)
- [Graham Etherington](https://github.com/ethering)
- [Fidel Ramirez](https://github.com/fidelram)
- [Daniel Blankenberg](https://github.com/blankenberg)
- [James Taylor](https://github.com/jxtx)
- [Bjoern Gruening](https://github.com/bgruening)


