---
site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
title: 'Variant Analysis of SARS-CoV-2 Sequencing Data'
tags: [galaxy, COVID-19, tools, workflows]
---

With the recent appearance of novel SARS-CoV-2 lineages in the UK, South Africa
and Brazil, which are suspected to display increased infectivity and/or partial
escape from neutralizing antibodies, the need for continuous tracking of
viral evolution through genome sequencing has received increased public
attention.

While the UK's COVID-19 Genomics Consortium (COG-UK) has been pioneering such
monitoring since early on in the pandemic, other countries are now eager to
establish their own networks for genomic surveillance.

In the framework of covid19.galaxyproject.org, an international team of Galaxy
developers and researchers from different disciplines (including members of the
Galaxy EU team and of the virology department of the University of Freiburg)
have collaborated since February last year to establish a set of workflows for
SARS-CoV-2 genomics analysis - including the ones that are urgently needed now.

The European Galaxy server offers these as
[__public workflows__](https://usegalaxy.eu/workflows/list_published).
Just search the list for *covid19.galaxyproject.org* and you will find our
recommended, best-practice workflows for SARS-CoV-2 variant analysis from
a variety of sequencing data sources: be it whole-genome sequencing or
ampliconic (ARTIC) data sequenced using short-read (Illumina, single-end or
paired-end) or long-read (ONT) data, we have workflows for calling variants
(even low-frequency intra-sample ones), generating standardized
technology-independent reports, and for building reliable consensus sequences!

Combine, *e.g.*, our
[__ARTIC Illumina PE variant calling__](https://usegalaxy.eu/u/wolfgang-maier/w/covid19-variation-analysis-on-artic-pe)
workflow with the workflow for
[__consensus sequence building__](https://usegalaxy.eu/u/wolfgang-maier/w/covid-19-consensus-construction)
and you have an analysis pipeline for analyzing typical diagnostic batches of
data, like most of the data generated by COG-UK, and for turning the results
into a set of consensus sequences ready for submission to
[GISAID](https://www.gisaid.org/) or for
[clade assignment with Nextstrain](https://clades.nextstrain.org/).

All these workflows have been built and tested specifically with scalability in
mind: on usegalaxy.eu we have verified, using large batches of COG-UK and other
data, that they let you analyze even thousands of samples in parallel.
